authenticity-collector/.env.example

```diff
@@ -1,2 +1,6 @@
 DATABASE_URL=<DATABASE_URL>
-PROVE_OF_OWNERSHIP_URL=<URL>
\ No newline at end of file
+PROVE_OF_OWNERSHIP_URL=<URL>
+PORT=<PORT THE APP IS LISTEN TO>
+API_URL=<IS_URL>
+API_KEY=<API_KEY>
```

Comment:
If this is the url of the integration service please name it IS_API_KEY and IS_API_URL